app/models/comment.rb

```diff
@@ -1,8 +1,12 @@
 class Comment < ApplicationRecord
+  TITLE_DELETED = "[deleted]".freeze
+  TITLE_HIDDEN = "[hidden by post author]".freeze
```

Comment:
refactored this so if we ever change it in the frontend website, it changes in the API as well

Reply:
This is why always advocate for using I18n, even in apps that only use one language. Much nicer also for specs, where you can assert on presence of `I18n.t(...)`, so copy can change without unnecessary breakage.

Reply:
Agreed, at some point we should start moving strings outside the code as much as possible :)

We do have a few i18n/l10n issues: https://github.com/thepracticaldev/dev.to/labels/area%3A%20internationalization%2Flocalization